ostelco-ios-client/ViewControllers/Home/HomeViewController.swift

```diff
@@ -149,11 +153,16 @@ class HomeViewController: ApplePayViewController {
             }
             .catch { error in
                 ApplicationErrors.log(error)
-            }
+        }
+    }
+
+    @objc func didPullToRefresh() {
+        hideMessages()
+        refreshBalance()
     }
 
     @IBAction private func buyDataTapped(_ sender: Any) {
-       if hasSubscription {
+        if hasSubscription {
```

Comment:
Is this whitespace change intentional to fix misalignment, or did this sneak in?

Reply:
intentional